serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -32,6 +32,7 @@ metadata:
     sw.kogito.kie.org/profile: dev <1>
 spec: <2>
   flow:
+    specVersion: 0.8.0
```

Comment:
This is a default value on the k8s api. Unfortunately some old versions of the server and kubectl won't replace it with the default value. We need to add a note here about this fix on newer kubectl versions. It's on my plate.